dpks/quant_matrix.py

```diff
@@ -438,29 +419,45 @@ def compare_groups(
 
         return self
 
-    def impute(self, method: str, **kwargs: int) -> QuantMatrix:
+    def classify(
+        self,
+        classifier,
+        shap_algorithm: str = "auto",
+        scale: bool = True,
+        rfe_step: int = 1,
+        rfe_min_features_to_select: int = 1,
+    ) -> QuantMatrix:
+        le = LabelEncoder()
+        Y = le.fit_transform(self.quantitative_data.var["group"].values)
+        X = self.quantitative_data.X.copy().transpose()
```

Comment:
I wonder now if we should have a check that will only use proteins/features that were considered in differential expression, or a limit of measurements per group? This could be added as a parameter to the QuantMatrix API call here. 

This works for now, but could be something we want to keep in mind to add in the future

Reply:
I implemented the limit per group